app/master/build_request.py

```diff
@@ -22,6 +22,14 @@ class BuildRequest(object):
         "job_name": "clusterrunner_configured_job",
```

Comment:
I'm actually more worried about the 'job_name' field in the request param, more than the name field in the JobConfig, as that might be the value that some dashboards are utilizing.

That's something that should be fixed though, if it does that.